designsafe/apps/accounts/views.py

```diff
@@ -332,6 +332,11 @@ def profile_edit(request):
     if request.method == 'POST':
         form = forms.UserProfileForm(request.POST, initial=tas_user)
         if form.is_valid() and pro_form.is_valid():
+            if (tas.get_user(email=form.cleaned_data['email']) and
```

Comment:
This code could be rewritten to make it a little more explicit that we're checking for a different user with the same email. Something like:
```suggestion
            existing_user = (tas.get_user(email=form.cleaned_data['email'])
            if existing_user and existing_user['username'] != tas_user['username]:
                # error logic
```

Reply:
@jarosenb  Cool!